flex/views/experiment/TabSeqSetupView.mxml

```diff
@@ -443,16 +443,16 @@
 				<mx:VBox width="100%" verticalGap="0" paddingLeft="2">
 					<mx:RadioButtonGroup id="seqPrepRadioGroup"/>
 					<mx:HBox id="hbox2" width="100%">
-						<mx:RadioButton id="samplesToBePreppedCheckbox"  value="Y" groupName="seqPrepRadioGroup"  label="Yes, please prep library" selected="true" click="{pickSeqPrepState()}">
+						<mx:RadioButton id="samplesToBePreppedCheckbox"  value="Y" groupName="seqPrepRadioGroup"  label="Yes, please prep library" selected="false" click="{pickSeqPrepState()}">
 						</mx:RadioButton>
 					</mx:HBox>
 
 					<mx:HBox >
-						<mx:RadioButton id="samplesAlreadyPreppedCheckbox"  value="N" groupName="seqPrepRadioGroup" label="No, library has already been prepped "
+						<mx:RadioButton id="samplesAlreadyPreppedCheckbox"  value="Y" groupName="seqPrepRadioGroup" label="No, library has already been prepped "
```

Comment:
Hey John -

Here you have set both of your radio group buttons for this group to a value of 'Y'.

Matt caught this and is going to check in the fix.

Thanks.